docs/sources/next/javascript-api/k6-experimental/browser/_index.md

```diff
@@ -109,41 +110,44 @@ C:\k6> k6 run script.js
+| K6_BROWSER_ARGS                | Extra command line arguments to include when launching browser process. See [this link](https://peter.sh/experiments/chromium-command-line-switches/) for a list of Chromium arguments. Note that arguments should not start with `--` (see the [example](#example)).                                                                                                                    |
+| K6_BROWSER_DEBUG               | All CDP messages and internal fine grained logs will be logged if set to `true`.                                                                                                                                                                                                                                                                                                         |
+| K6_BROWSER_EXECUTABLE_PATH     | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                                                                                                                                              |
+| K6_BROWSER_HEADLESS            | Show browser GUI or not. `true` by default.                                                                                                                                                                                                                                                                                                                                              |
+| K6_BROWSER_IGNORE_DEFAULT_ARGS | Ignore any of the [default arguments](#default-arguments) included when launching a browser process.                                                                                                                                                                                                                                                                                     |
+| K6_BROWSER_TIMEOUT             | Default timeout to use for various actions and navigation. `'30s'` if not set.                                                                                                                                                                                                                                                                                                           |
+| K6_BROWSER_TRACES_METADATA     | Sets additional _key-value_ metadata that will be included as attributes in every span generated from browser module traces. Example: `K6_BROWSER_TRACES_METADATA=attr1=val1,attr2=val2`. This will only apply if traces generation is enabled by k6 options, see `--traces-output` option [docs](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/reference#traces-output). |
```

Comment:
```suggestion
| K6_BROWSER_TRACES_METADATA     | Sets additional _key-value_ metadata that is included as attributes in every span generated from browser module traces. Example: `K6_BROWSER_TRACES_METADATA=attr1=val1,attr2=val2`. This only applies if traces generation is enabled, refer to [Traces output](https://grafana.com/docs/k6/<K6_VERSION>/using-k6/k6-options/reference#traces-output) for more details. |
```

Reply:
Is this `<K6_VERSION>` automatically replaced?

Reply:
@inancgumus yep, it gets replaced to `latest` or whichever version the user has selected at the moment.

Reply:
@heitortsergent Wow, that's awesome!

Reply:
:+1: Changed in: [ae9bea9](https://github.com/grafana/k6-docs/pull/1455/commits/ae9bea962278552358f03465bcfbd187a1def486)